src/commands/synthetics/run-tests-command.ts

```diff
@@ -303,6 +314,16 @@ export class RunTestsCommand extends Command {
         variables: parseVariablesFromCli(this.variableStrings, (log) => this.reporter.log(log)),
         pollingTimeout:
           this.pollingTimeout ?? this.config.defaultTestOverrides?.pollingTimeout ?? this.config.pollingTimeout,
+        allowInsecureCertificates: validatedOverrides.allowInsecureCertificates,
+        body: validatedOverrides.body,
+        bodyType: validatedOverrides.bodyType,
+        defaultStepTimeout: validatedOverrides.defaultStepTimeout,
+        executionRule: validatedOverrides.executionRule,
+        followRedirects: validatedOverrides.followRedirects,
+        retry: Object.keys(cliOverrideRetryConfig).length > 0 ? cliOverrideRetryConfig : undefined,
+        startUrl: validatedOverrides.startUrl,
+        startUrlSubstitutionRegex: validatedOverrides.startUrlSubstitutionRegex,
+        testTimeout: validatedOverrides.testTimeout,
```

Comment:
(nit) Can you order these alphabetically? Also reminding you about the similar comment from the previous PR ;)